src/driver/op.rs

```diff
@@ -103,7 +119,7 @@ where
     }
 }
 
-impl<T> Future for Op<T>
+impl<T> Future for Op<T, SingleCQE>
```

Comment:
Constrained this implementation to SingleCQE types, which includes all Ops in the current implementation